tests/test_plugins_apps_remoteaccess_teamviewer.py

```diff
@@ -8,7 +8,7 @@
 def test_teamviewer_plugin_global_log(target_win_users, fs_win):
 
     teamviewer_logfile = absolute_path("data/TestTeamviewer.log")
-    target_logfile_name = "sysvol/Program Files/TeamViewer/TestTeamviewer.log"
+    target_logfile_name = "/sysvol/Program Files/TeamViewer/TestTeamviewer.log"
```

Comment:
I think I missed it but why does this matter now?

Reply:
Before it depended on the whether the `path` parameter given to `RootFilesystem.get()` had a leading `/` or not on whether the `path` attribute of the `RootFilesystemEntry` would have one or not. Now it is always consistently the same: always a leading `/`. Added to that, it means that the code flow in `RootFilesystem.get()` can be more similar to `VirtualFilesystem.get()`.

Reply:
What happens if you don't add a leading `/` now?

Reply:
It will be added to the `RootFilesystemEntry`'s `.path` attribute no matter what.